gradle.properties

```diff
@@ -36,7 +36,7 @@ bouncycastleVersion=1.64
 hadoopVersion=2.7.1
 postgreSQLVersion=42.2.5
 httpClientVersion=4.5.10
-cmClientVersion=7.1.1
+cmClientVersion=7.1.4
```

Comment:
Should we go to a much newer released version? My worry is that if it comes from a private repo they might delete it at some point?

Reply:
would be nice but https://jira.cloudera.com/browse/RELENG-13626